### PR TITLE
Lost or stolen email

### PR DESCRIPTION
**Pull request by easternbloc**
New template for lost-stolen

### CODE REVIEW

File: controllers/check-details.js [hunk 1]
```diff
@@ -22,8 +31,9 @@ Submit.prototype.saveValues = function saveValues(req, res, callback) {
   var data = _.pick(req.sessionModel.toJSON(), _.identity);
   var model = new Model(data);
 
-  if (templateMap[req.originalUrl]) {
-    model.set('template', templateMap[req.originalUrl]);
+  if (serviceMap[req.originalUrl]) {
+    model.set('template', serviceMap[req.originalUrl].template);
+    model.set('subject', serviceMap[req.originalUrl].subject);
   } else {
     throw new Error('no template found');
```
Comment (JoeChapman) [inline controllers/check-details.js]: this message is no longer accurate, wha d'y think?

File: services/email/index.js [hunk 2]
```diff
@@ -11,13 +11,13 @@ var path = require('path');
 
 var htmlTemplates = {
   error: fs.readFileSync(path.resolve(__dirname, './templates/error_html.mus')).toString('utf8'),
-  lostOrStolen: fs.readFileSync(path.resolve(__dirname, './templates/error_html.mus')).toString('utf8'),
+  'lost-or-stolen': fs.readFileSync(path.resolve(__dirname, './templates/lost_or_stolen_html.mus')).toString('utf8'),
```
Comment (JoeChapman) [inline services/email/index.js]: :+1:

File: services/email/index.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ Emailer.prototype.send = function send(emailConfig, callback) {
   this.transporter.sendMail({
     from: config.email.from,
     to: emailConfig.to,
-    subject: 'BRP Card',
+    subject: emailConfig.subject,
```
Comment (JoeChapman) [inline services/email/index.js]: `emailConfig` or `config.email`?

**Comment by JoeChapman** [comment]
@easternbloc how should I test this, can it be tested?

File: controllers/check-details.js (hunk 1 above)
Comment (easternbloc) [inline controllers/check-details.js]: Agreed. Will update

File: services/email/index.js (hunk 3 above)
Comment (easternbloc) [inline services/email/index.js]: Perhaps need to reconsder naming. config.email is static or from envs. emailConfig was passed. I'll update it.

**Pull request merged**